swagger/ghc.yaml

```
@@ -1208,6 +1208,45 @@ paths:
           description: A server error occurred
           schema:
             $ref: '#/responses/ServerError'
+  '/tac/valid':
+      get:
+        summary: Validation of a TAC value
+        description: Returns a boolean based on whether a tac value is valid or not
+        operationId: tacValidation
+        tags:
+          - tac
+          - moveOrder
+        parameters:
+          - in: query
+            name: tac
+            type: string
```

I wasn't keenly following all the discussion here but were you interested in setting a min or max length validation here?

---

I think for this api parameter, we won't limit the length of the string since we want to return `{isValid: true}` or `{isValid: false}` and don't want to return an error.  But that is a good point to set the value to 4 digits in the updateMoveOrder payload.  There's some questions in the air, so we'll take care of that in another story.